rascal-lsp/src/main/rascal/util/LanguageServer.rsc

```diff
@@ -139,16 +139,15 @@ data Command(str title="")
 
 data InlayHint
     = hint(
-        loc range, // should point to a cursor position (begin and end column same)
+        loc position, // should point to a cursor position (begin and end column same)
```

Comment:
this constraint affects all DSL implementations, such that they have to make their start positions equal to their end. I propose we use the start position of _any_ location instead and solve this constraint under-the-hood.

Reply:
yeah, I should word this differently. The comment was more right than the name of the variable. I'll update the comment a bit. but true, what we do is we put it at the start of the location.

Reply:
I've changed it, this is better to use. thanks for helping not just follow LSP spec ;)

Reply:
that's what I'm here for ;-)